test/metagrammars/good/rule_num_range

```diff
@@ -0,0 +1,7 @@
+:components:
+onebyte := 17-250
+twobyte := 1-280
+eightbit := b1-b01101100
+twobit := b0-b10
+
+root := onebyte{2}:twobyte:eightbit
```

Comment:
This doesn't exercise the case of matching on a byte-specified value range, like `0x40-0x50` in the PR description. That should be addressed before merging.